### PR TITLE
Update type of useQuery

### PR DESCRIPTION
**Pull request by Bricks666**
Update package for correct using with @farfetched/core >=0.3.0

### CODE REVIEW

**Comment by Bricks666** [comment]
Update package for correct using with @farfetched/core >=0.3.0

**Comment by Bricks666** [comment]
@igorkamyshev

File: .changeset/brave-rats-fetch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@farfetched/react': minor
```
Comment (igorkamyshev) [inline .changeset/brave-rats-fetch.md:2]: It is not minor, it is a fix of a bug. So, it is patch 🤗

```suggestion
'@farfetched/react': patch
```

File: packages/react/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@farfetched/react",
-  "version": "0.2.1",
+  "version": "0.3.0",
```
Comment (igorkamyshev) [inline packages/react/package.json:3]: Repository generates versions automatically, check README.md for more detailed development flow description. So, please revert this change.

File: packages/react/src/use_query.ts [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import { type Query } from '@farfetched/core';
 import { useUnit } from 'effector-react';
 
-function useQuery<Params, Data, Error>(
-  query: Query<Params, Data, Error>
+function useQuery<Params, Data, Error, InitialData>(
```
Comment (igorkamyshev) [inline packages/react/src/use_query.ts:4]: I think, this is the case for type tests 🤗

**Review by igorkamyshev — CHANGES_REQUESTED** [review]
Hello! Thanks for the fix 🤗

Could you add type_spec file for your fix to guarantee that the problem was there, and it is fixed now? All Farfetched PRs have to be tested before merge — https://farfetched.pages.dev/statements/tests.html

File: .changeset/brave-rats-fetch.md (hunk 1 above)
Comment (igorkamyshev) [reply .changeset/brave-rats-fetch.md:2]: So, now it is two changes 🤗 old one and new one. I assume, we can delete one of them.

**Review by igorkamyshev — CHANGES_REQUESTED** [review]

**Comment by Bricks666** [comment]
Oh sorry, didn't notice. I fixed it

**Review by igorkamyshev — APPROVED** [review]
Great, thanks 🤗

It'll be released in a couple of days with farfetched@0.4

**Pull request merged**